components/badge/badge_input.c

```diff
@@ -76,6 +77,7 @@ badge_input_add_event(uint32_t button_id, bool pressed, bool in_isr)
 				ets_printf("badge_input: input queue full.\n");
 			}
 		}
+		badge_vibrator_buzz(70000);
```

Comment:
The badge_input_add_event() is sometimes called from within an interrupt. (e.g. the flash button on the rev 1.0.0)

We're not allowed to do I2C from an interrupt. This will result in a crash.